Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -634,6 +638,10 @@ def SwapKeepAndDrop(l):
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_MergedTrackTruth_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_StripDigiSimLink_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_PixelDigiSimLink_*')
+approxSiStripClusters.toModify(FEVTDEBUGHLTEventContent,
```

Comment:
Do we really need this line?
`FEVTDEBUGHLT` already calls `FEVTDEBUG` with
`FEVTDEBUGHLTEventContent.outputCommands.extend(FEVTDEBUGEventContent.outputCommands)`

https://github.com/cms-sw/cmssw/blob/master/Configuration/EventContent/python/EventContent_cff.py#L632

Reply:
Yes, it's needed b/c there is a drop *_hlt*_*_* hidden in one of the append statements.